fastapi-challenges/websocket-chat/devashmit-day19/main.py

```
@@ -0,0 +1,57 @@
+    async def connect(self,room,ws,name):
+        await ws.accept()
+        if room not in self.rooms:self.rooms[room]=[]
+        self.rooms[room].append((ws,name))
+        await self.broadcast(room,f"🟢 {name} joined",exclude=ws)
+    async def disconnect(self,room,ws,name):
+        if room in self.rooms:
+            self.rooms[room]=[(w,u) for w,u in self.rooms[room] if w is not ws]
+            if not self.rooms[room]:del self.rooms[room]
+        await self.broadcast(room,f"🔴 {name} left")
+    async def broadcast(self,room,msg,exclude=None):
+        if room not in self.rooms:return
```

Ruff (0.15.17)

[error] Multiple statements on one line (colon)

(E701)